play/src/front/Components/Tooltip/HelpTooltip.svelte

```diff
@@ -25,7 +27,14 @@
     />
     <div class="relative z-10 pb-4 rounded-lg overflow-hidden">
         {#if hasImage}
-            <img alt="Help GIF" src={image} class="w-full -mt-[2px]" loading="lazy" />
+            {#if helpVideo}
+                <video autoplay muted loop class="w-full -mt-[2px]">
+                    <source src={media} type="video/mp4" />
+                    Your browser does not support the video tag.
+                </video>
+            {:else}
+                <img alt="Help GIF" src={media} class="w-full -mt-[2px]" loading="lazy" />
+            {/if}
```

Comment:
I'm not a huge fan of the helpVideo boolean flag.

2 solutions:

- Either add 2 different properties (`helpVideo` VS `helpImage`)
- Or only one `helpMedia` and switch to video if the `helpMedia` URL ends with ".mp4".